app/helpers/application_helper.rb

```diff
@@ -1,4 +1,12 @@
 module ApplicationHelper
   include ::SufiaHelper
   include ContentBlockHelper
+
+  def nav_link(href)
```

Comment:
Agreed. I didn't do it in sufia both because it only has 3 items right now and such a change probably should happen down in curation_concerns anyway.